northpoint-development/Vector-Tiles-Google-Maps
# BMAPS-1694 Added @mapbox/vector-tile dependency

## Issue Link
[BMAPS-1694](https://northpoint-development.atlassian.net/browse/BMAPS-1694)

## Description
- This does not change any functionality, but rather, removes three files: `VectorTile.js`, `VectorTileFeature.js`, and `VectorTileLayer.js`.
- Inside `MVTSource.js`, imported `VectorTile` from the `@mapbox/vector-tile` package, which has a lot of the functionality the three removed files contained
- Also inside `MVTSource.js`, added a for-loop for parsing the geometry of each feature first so it can be drawn; this code came from the `VectorTile.js` file that was removed

**NOTE:** No tests were added, this just changes where the `new VectorTile()` code comes from in `MVTSource.js`  

## QA (By code reviewer)
- Pull down this branch
- Inside the `basic.html` example, change the Google Maps api key to be the key we use in `index.html` of Beyond Maps
- Open the `basic.html` example using Live Server and ensure everything loads properly

[BMAPS-1694]: https://northpoint-development.atlassian.net/browse/BMAPS-1694?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
